# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/netkit-telnet/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- netkit-telnet-0.17/telnetd/utility.c.vanilla	2024-10-27 14:22:38.496649006 +0100
+++ netkit-telnet-0.17/telnetd/utility.c	2024-10-27 14:24:25.056647544 +0100
@@ -40,6 +40,7 @@
 #define PRINTOPTIONS
 
 #include <stdarg.h>
+#include <time.h>
 #include <sys/utsname.h>
 
 #ifdef AUTHENTICATE
